tests/inference/ebm_test.py

```diff
@@ -15,6 +15,8 @@
 """Tests for qhbmlib.inference.ebm"""
 
 import functools
+import itertools
+import time
```

Comment:
unused?

Reply:
Removed.  Noted the linter is not catching these in #126